development/compiling/compiling_for_android.rst

```diff
@@ -8,147 +8,154 @@ Compiling for Android
 Note
 ----
 
-For most cases, using the built-in deployer and export templates is good
+In most cases, using the built-in deployer and export templates is good
 enough. Compiling the Android APK manually is mostly useful for custom
 builds or custom packages for the deployer.
 
-Also, you still need to do all the steps mentioned in the
-:ref:`doc_exporting_for_android` tutorial before attempting your custom
-export template.
+Also, you still need to follow the steps mentioned in the
+:ref:`doc_exporting_for_android` tutorial before attempting to build
+a custom export template.
 
 Requirements
 ------------
 
 For compiling under Windows, Linux or macOS, the following is required:
 
--  Python 2.7+ or Python 3.5+
--  SCons build system
--  [Windows only] PyWin32 (optional, for parallel compilation)
--  Android SDK version 23.0.3 [Note: Please install all tools and extras of the SDK Manager]
--  Android build tools version 19.1
--  Android NDK r13 or later
+-  `Python 2.7+ or Python 3.5+ <https://www.python.org/downloads/>`_
+-  `SCons <https://scons.org/pages/download.html>`_ build system
+-  `Android SDK <https://developer.android.com/studio/#command-tools>`_ (command-line tools are sufficient)
```

Comment:
What exact component is needed should likely be made clearer, users are often confused about it. In my experience at least the "Android SDK Tools", "Android SDK Platform-tools", latest (or target from manifest?) "Android SDK Build-tools" and "Android Support Repository".

I'll try again from scratch to see what it wants now.

Reply:
I tried to set things up anew to document the exact steps needed. Here's what I had to do on Linux:
```
21:26 <Akien> - Download command line tools from https://developer.android.com/studio/#command-tools
21:26 <Akien> - Unzip in the root of the (empty) ANDROID_SDK_HOME, gives the "tools" folder
21:27 <Akien> - Run `./tools/bin/sdkmanager --licenses` to accept licenses
21:27 <Akien> Then gradlew will handle the rest
```
The step for accepting licenses is not straightforward, so IMO it would be worth being explicit about it. Maybe not in this list of requirements as it would quickly get crowded, but maybe a paragraph after the list (and here a "See below for installation details.").